examples/kubernetes/README.md

```diff
@@ -21,7 +21,7 @@ You'll need [Go 1.3+](http://golang.org/doc/install) in order to build the
 ### Build and install vtctlclient
 
 ```
-$ go get github.com/youtube/vitess/go/cmd/vtctlclient
+vitess/examples/kubernetes$ go build ../../go/cmd/vtctlclient
```

Comment:
Why the change? The "go get" one still works for me in a clean Docker image, and I think it's more self-contained because you don't have to download anything in advance.